pkg/pods/pod.go

```diff
@@ -563,6 +585,17 @@ func (pod *Pod) setupConfig(manifest manifest.Manifest, launchables []launch.Lau
 		return util.Errorf("Error writing platform config file for pod %s: %s", manifest.ID(), err)
 	}
 
+	resourceLimitFilename, err := manifest.ResourceLimitsConfigFileName()
```

Comment:
Do we need to support manifests that don't have this set for the time being?

Reply:
Good point. I tested it with and without the manifest stanza and it worked, but I'll double checked that it's doing exactly what we want it to.